crates/aptos-crypto/src/bls12381/bls12381_keys.rs

```diff
@@ -209,6 +211,8 @@ impl VerifyingKey for PublicKey {
 }
 
 impl ValidCryptoMaterial for PublicKey {
+    const AIP_80_PREFIX: &'static str = "bls12381-pub-";
```

Comment:
I am a bit confused: AIP-80 says it only "standardizes private keys."

 > This AIP defines how private keys should be represented off-chain.

Reading a bit closely, it also seems to (not sure) propose to add a `pub` (sub)prefix to public keys.

Seeing this now, I am a bit worried about changing serialization formats of public keys. For example, the format here is used to serialize validator pubkeys on chain, so backwards compatibility issues would be a big concern.

Can you clarify in the PR description why this won't break anything? It seems to suggest that only the CLI will leverage these new formats?

Reply:
This does not change serialization formats of keys